src/main/java/uk/gov/hmcts/darts/notification/entity/Notification.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.darts.notification.entity;
```

Comment:
Wondering should we rename this to be `entities` to match the package under common or vice-versa.
build.gradle stuff too

Reply:
I have it in my head that package should always be singular, but I think it is again down to personal perference. Either way, we should be consistent.
@jackmaloney another question for you if you have a preference?

Reply:
in fact, been looking at another project in hmcts, and they seem to be mostly singular also, so guess go with that.
![image](https://user-images.githubusercontent.com/131666106/234838917-b18cf58a-4799-4e77-92af-56a416dbcb5f.png)

i'll change entities to entity in common